sql/core/src/test/resources/sql-tests/results/timestampNTZ/datetime.sql.out

```diff
@@ -1222,26 +1222,26 @@ select to_timestamp('2019-10-06 A', 'yyyy-MM-dd GGGGG')
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.SparkUpgradeException
-You may get a different result due to the upgrading of Spark 3.0: Fail to recognize 'yyyy-MM-dd GGGGG' pattern in the DateTimeFormatter. 1) You can set spark.sql.legacy.timeParserPolicy to LEGACY to restore the behavior before Spark 3.0. 2) You can form a valid datetime pattern with the guide from https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html
+java.lang.RuntimeException
```

Comment:
Is it intentional replacement?

Reply:
Yes, it is from https://github.com/apache/spark/pull/33019. Do you think we should use another exception type?

Reply:
I am ok with the exception type.